src/resources/js/views/accordion.js

```diff
@@ -55,7 +55,7 @@ tribe.events.views.accordion = {};
 		$container.find( obj.selectors.accordionTrigger ).each( function( index, header ) {
 			var $header = $( header );
 			var contentId = $header.attr( 'aria-controls' );
```

Comment:
Could it be just a selector? Not ID? Making it available to use classes too.

Reply:
An accordion header button controls the accordion contents. They are linked by an id (`aria-controls` attribute is the `id` of the contents that the header controls). So there's no other good way of getting the contents that are linked to the header.

Reply:
Could we link them via a selector instead of an ID?

Reply:
not likely. if a page has multiple accordions with the same markup, we need to know which header controls which content. the accordion.js is meant to be extendable for any element.

if you're thinking of something else, would love to chat it out to see if we can balance both.

Reply:
Sounds good.